server/src/js/core.js

```diff
@@ -65,8 +58,7 @@ window.flood = {
           e.preventDefault()
           window.history.back()
         })
-        // ie 11 prepend hack
-        nav.insertBefore(hyperlink, nav.childNodes[0])
+        container.appendChild(hyperlink)
```

Comment:
Does this work on IE11, the code removed says ie 11 prepend hack, i know new code doesn't use prepend but needs checking.

Reply:
Also how come core functionality for the back button is being updated for this ticket?

Reply:
This is because of a code smell from sonar cloud saying the back button didnt have an aria-label. so worked with Dan to edit this so it didnt need one.